lang/cs/Org.Apache.REEF.IO.Tests/TestFilePartitionInputDataSet.cs

```diff
@@ -173,7 +174,7 @@ public void TestWithByteDeserializer()
                         .GetInstance<IInputPartition<IEnumerable<byte>>>();
```

Comment:
I didn't want to break backward compatibility, so I opted to keep `T` as `IEnumerable`. We will be writing a new version of this class, and the test will be revised.